emperor/support_files/js/util.js

```diff
@@ -48,14 +48,13 @@ var k_CHROMABREWER_MAPNAMES = ['Classic QIIME Colors',
 /**
  *
  * Sorting function that deals with alpha and numeric elements
- * 
+ *
```

Comment:
It seems like you modified the docstring for this function by accident, would you mind undoing the changes?